core/tauri-codegen/src/context.rs

```diff
@@ -125,6 +125,16 @@ enum Target {
   Ios,
 }
 
+impl Target {
+  fn mobile(&self) -> bool {
+    matches!(self, Target::Android | Target::Ios)
+  }
+
+  fn desktop(&self) -> bool {
+    !self.mobile()
+  }
```

Comment:
Not 100% sure we've been following this convention throughout the codebase, but since these aren't returning a "field" on the item but a property of it we may want to use the `is_` prefix.

```suggestion
  fn is_mobile(&self) -> bool {
    matches!(self, Target::Android | Target::Ios)
  }

  fn is_desktop(&self) -> bool {
    !self.mobile()
  }
```